_sass/_typography.scss

```diff
@@ -0,0 +1,27 @@
+article.post.typography{
```

Comment:
Avoid qualifying class selectors with an element.<br>Opening curly brace `{` should be preceded by one space